ConcernsCaseWork/ConcernsCaseWork/Security/RbacManager.cs

```diff
@@ -13,44 +16,23 @@ namespace ConcernsCaseWork.Security
 	/// </summary>
 	public sealed class RbacManager : IRbacManager
 	{
-		private readonly IUserRoleCachedService _userRoleCachedService;
 		private readonly ILogger<RbacManager> _logger;
+		private readonly ITeamsCachedService _teamsService;
 
-		private readonly string[] _defaultUsers;
 
-		public RbacManager(IConfiguration configuration, IUserRoleCachedService userRoleCachedService, ILogger<RbacManager> logger)
+		public RbacManager(ILogger<RbacManager> logger, ITeamsCachedService teamsService)
 		{
-			_userRoleCachedService = userRoleCachedService;
-			_logger = logger;
-			
-			// set hardcoded users from configuration
-			// update / remove when AD integration
-			_defaultUsers = configuration["app:username"].Split(',');
+			_logger = Guard.Against.Null(logger);
+			_teamsService = Guard.Against.Null(teamsService);
 		}
 
-		public Task<IList<string>> GetSystemUsers(params string[] excludes)
-		{			
-			return Task.FromResult(
-				UserRoleMap.GetDefaultUsersExcludeE2E(_defaultUsers.Where(x => !excludes.Contains(x)))			
-			);
-		}
-		
-		public async Task<IDictionary<string, RoleClaimWrapper>> GetUsersRoles()
-		{
-			_logger.LogInformation("RbacManager::GetUsersRoles");
-			
-			var usersRoleClaim = await _userRoleCachedService.GetUsersRoleClaim(_defaultUsers);
-			
-			return new SortedDictionary<string, RoleClaimWrapper>(usersRoleClaim);
-		}
-
-		public async Task<RoleClaimWrapper> GetUserRoleClaimWrapper(string user)
+		public async Task<IList<string>> GetSystemUsers(params string[] excludes)
 		{
-			_logger.LogInformation("RbacManager::GetUserRoleClaimWrapper {User}", user);
+			// TODO: Integrate the known users from the DB with Azure graph to build up a set of users where we can identify those who aren't in the graph and may have left.
 
-			var roleClaimWrapper = await _userRoleCachedService.GetRoleClaimWrapper(_defaultUsers, user);
-			
-			return roleClaimWrapper;
+			_logger.LogMethodEntered();
+			var users = await _teamsService.GetTeamOwners();
```

Comment:
This is probably just me misunderstanding, but the method is called Get System Users (is that the users of the system, or system-level users? I guess the former). But it returns Team Owners. Is that all users? Or just a certain type of user? (I'm not sure if everyone is a team owner).  I'm also a bit unclear of the format of what goes into 'excludes' or the reason for it. Again, probably something that would be clearer when I've got more familiarity with the code, but thought I'd mention it.

Reply:
That is all users. We are basically re-using the team owners as users who have been seen by the system.
(when they log in we are creating an empty team). 
We could do with refactoring it somehow but timescales are too tight.
So I hid this inside the RbacManager.